Packs/CheckpointFirewall/Integrations/CheckPoint_FW.py

```diff
@@ -0,0 +1,1152 @@
+from typing import Tuple
+import requests
+
+import demistomock as demisto
```

Comment:
The files are not in the correct folder.
It should be `Packs/CheckpointFirewall/Integrations/CheckPointFirewall_v2/CheckPointFirewall.py`
And change the name of the integration files to be `CheckPointFirewall_v2`